File: GettingStarted/allPrime.cpp
#include <iostream>
using namespace std;

bool isPrime_(int n){ 
    bool prime=true;
    for(int i=2; i * i <= n ; i++){
        if(n%i==0){
            prime=false;
            break;
            
        }
    }
    return prime;
}

void printAllPrime(int a,  int b){
        for(int i=a; i<=b ; i++){
            bool ans= isPrime_(i);
            if(ans) cout << i << endl;
            
        }
    }
int main(int argc, char **argv){
    int low, high;
    cin >> low >> high;
    printAllPrime(low, high);
    
}

File: GettingStarted/allPrime.java
import java.util.Scanner;


public class Main{
    public static Scanner scn = new Scanner(System.in);
    public static boolean isPrime_(int n){ 
        boolean prime=true;
        for(int i=2; i * i <= n ; i++){
            if(n%i==0){
                prime=false;
                break;
                    
            }
                
        }return prime;
            
    }
    public static void printAllPrime(int a,  int b){
        for(int i=a; i<=b ; i++){
            boolean ans= isPrime_(i);
            if(ans) System.out.println(i);
                
        }
            
    }
    public static void main(String[] args){
        int low = scn.nextInt();
        int high = scn.nextInt();
        printAllPrime(low, high);
            
    }
        
}